pkg/cluster/cluster_config.go

```diff
@@ -120,3 +121,31 @@ func GetPlatform(cli client.Client) (Platform, error) {
 	// check and return whether ODH or self-managed platform
 	return isSelfManaged(cli)
 }
+
+type Release struct {
+	Name    Platform                `json:"name,omitempty"`
+	Version version.OperatorVersion `json:"version,omitempty"`
+}
+
+func SetRelease(cli client.Client) (*Release, error) {
```

Comment:
May be it is GetRelease() since creates the structure and does not set it anywhere (and in sync with the error message).

Reply:
+1